app/celer-g4/RunInput.hh

```diff
@@ -63,6 +63,7 @@ struct RunInput
     // Physics setup options
     PhysicsListSelection physics_list{PhysicsListSelection::ftfp_bert};
     GeantPhysicsOptions physics_options;
+    bool disable_eloss_fluct{false};  //!< Forcibly disable eloss fluctuations
```

Comment:
How is this different from setting `GeantPhysicsOptions::eloss_fluctuation` to false? I thought we generally pulled this option through the EM parameters.

Reply:
That will only work with our geant physics list, not if we use FTFP BERT or something else... though you're right, for celer-sim it should be equivalent. Do you think we'd ever even need the original behavior where we don't allow fluctuations with a field, or should I just revert this? Now I'm not even sure why we were dong that in the first place instead of enabling/disabling them based on the imported EM parameters.

Reply:
I think the capability predated the parameters. I don't think there's a reason to disable if the corresponding Geant option is enabled: we really only wanted to do this for comparison with adept and for performance testing, and for both those cases the celer sim option should be sufficient.